netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -124,6 +129,15 @@ public NettyServerBuilder channelType(Class<? extends ServerChannel> channelType
   }
 
   /**
+   * Specifies a channel option. As the underlying channel as well as network implementation may
+   * ignore this value applications should consider it a hint.
+   */
+  public <T> NettyServerBuilder withOption(ChannelOption<T> option, T value) {
```

Comment:
withChildOption?    These won't affect the listening socket itself.

Reply:
Thanks, Carl.  I will change the method name to withChildOption later.